src/map/script.c

```diff
@@ -16737,7 +16737,7 @@ BUILDIN(explode)
 	temp = aMalloc(len + 1);
 
 	for (i = 0; str[i] != '\0'; i++) {
-		if (str[i] == delimiter && (int64)start + k < (int64)(SCRIPT_MAX_ARRAYSIZE-1)) { // FIXME[Haru]: SCRIPT_MAX_ARRAYSIZE should really be unsigned (and INT32_MAX)
+		if (str[i] == delimiter && start + k < SCRIPT_MAX_ARRAYSIZE) {
```

Comment:
why you removed int64? Here probably possible overflow or underflow in ints
in other words what happened if here ``INT_MAX + INT_MAX < INT_MAX - 1``

Reply:
just initialize 100k element array already let the script engine stuck for 1 minute ....
hahaha fix something that is never meant to happen anyways :laughing: